static/css/settings/languages.css

```diff
@@ -0,0 +1,10 @@
+#languages tr .code{
+    width: 30%;
+    display: inline-block;
+    text-align: center;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6225754)